src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1727,6 +1727,10 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
         cmd.arg("--run-lib-path").arg(builder.sysroot_libdir(compiler, target));
         cmd.arg("--rustc-path").arg(builder.rustc(compiler));
 
+        // Minicore auxiliary lib for tests that need std/core stubs in cross-compilation scenarios.
```

Comment:
Nit: this is wrong "std/core" -> `core` only. Also should mention `no_core` + `check-pass` tests.

Reply:
Fixed.